pkg/services/authz/mappers/rbac_mapper.go

```diff
@@ -30,7 +30,7 @@ func NewK8sRbacMapper() *K8sRbacMapper {
 			utils.VerbUpdate:           fmt.Sprintf("%s:write", r),
 			utils.VerbPatch:            fmt.Sprintf("%s:write", r),
 			utils.VerbDelete:           fmt.Sprintf("%s:delete", r),
-			utils.VerbDeleteCollection: fmt.Sprintf("%s:delete", r),
+			utils.VerbDeleteCollection: fmt.Sprintf("%s:delete", r), // ????  SHOULD THIS something more??? %s.permissions:write
```

Comment:
@chaudyg @kalleep -- maybe I am overthinking this... in the end it would be translated to a bunch of Delete calls that will be checked individually anyway 🤔 

IIUC, it would internally List and then delete on each item after finalizers etc -- so the weird behavior is that we could delete a few things, then fail.  but that is OK